web_client/actions.jsx

```diff
@@ -557,6 +557,13 @@ export const resetUploadState = () => promiseAction(
   }
 );
 
+export const setAnalysisBusy = (busy=true) => promiseAction(
```

Comment:
Did the linter say anything about this? I expect it should ask you to put spaces around the `=` in `busy=true`.

EDIT: I see you fixed this in the style commit :-D